x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_detail.tsx

```diff
@@ -0,0 +1,342 @@
+    return [sectionData, displayDate];
+  }, [relatedEventToShowDetailsFor, naString]);
+
+  const waitCrumbs = useMemo(() => {
+    return [
+      {
+        text: eventsString,
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+    ];
+  }, [pushToQueryParams, eventsString]);
+
+  const crumbs = useMemo(() => {
+    return [
+      {
+        text: eventsString,
+        onClick: () => {
+          pushToQueryParams({ crumbId: '', crumbEvent: '' });
+        },
+      },
+      {
+        text: processName,
+        onClick: () => {
+          pushToQueryParams({ crumbId: processEntityId!, crumbEvent: '' });
+        },
+      },
+      {
+        text: (
+          <>
+            <EuiI18nNumber value={totalCount} />
+            {/* Non-breaking space->*/ ` ${eventsString}`}
+          </>
+        ),
+        onClick: () => {
+          pushToQueryParams({ crumbId: processEntityId!, crumbEvent: 'all' });
+        },
+      },
+      {
+        text: (
+          <>
+            <EuiI18nNumber value={countBySameCategory} />
+            {/* Non-breaking space->*/ ` ${relatedEventCategory}`}
+          </>
+        ),
+        onClick: () => {
+          pushToQueryParams({
+            crumbId: processEntityId!,
+            crumbEvent: relatedEventCategory || 'all',
+          });
+        },
+      },
+      {
+        text: relatedEventToShowDetailsFor
+          ? event.descriptiveName(relatedEventToShowDetailsFor)
+          : naString,
+        onClick: () => {},
+      },
+    ];
+  }, [
+    processName,
+    processEntityId,
+    eventsString,
+    pushToQueryParams,
+    totalCount,
+    countBySameCategory,
+    naString,
+    relatedEventCategory,
+    relatedEventToShowDetailsFor,
+  ]);
+
+  /**
+   * If the ship hasn't come in yet, wait on the dock
+   */
+  if (!relatedsReady) {
+    const waitingString = i18n.translate(
+      'xpack.securitySolution.enpoint.resolver.panel.relatedDetail.wait',
+      {
+        defaultMessage: 'Waiting For Events...',
+      }
+    );
+    return (
+      <>
+        <StyledBreadcrumbs breadcrumbs={waitCrumbs} />
+        <EuiSpacer size="l" />
+        <EuiTitle>
+          <h4>{waitingString}</h4>
+        </EuiTitle>
+      </>
+    );
+  }
+
+  /**
+   * Could happen if user e.g. loads a URL with a bad crumbEvent
+   */
+  if (!relatedEventToShowDetailsFor) {
+    const errString = i18n.translate(
+      'xpack.securitySolution.enpoint.resolver.panel.relatedDetail.missing',
+      {
+        defaultMessage: 'Related event not found.',
+      }
+    );
+    return (
+      <PanelContentError translatedErrorMessage={errString} pushToQueryParams={pushToQueryParams} />
+    );
+  }
+
+  return (
+    <>
+      <StyledBreadcrumbs truncate={false} breadcrumbs={crumbs} />
+      <EuiSpacer size="l" />
+      <EuiText size="s">
+        <BoldCode>{`${relatedEventCategory} ${event.ecsEventType(

```

Comment:
This also seems like a candidate for i18n:
```
<FormattedMessage
  id="nerds2"
  defaultMessage="{category} {eventType} @ {date}"
  values={/* object w/ values here */}
/>
```

Reply:
and here: https://github.com/elastic/kibana/pull/69098/commits/ee1f0e69e885e625bc62179d059a58d6116334f1